lib/releaselib.sh

```diff
@@ -639,6 +639,71 @@ release::gcs::copy_release_artifacts() {
 
 
 ###############################################################################
```

Comment:
This function is 80% similar to copy_release_artifacts.  We should reuse that and condition the bits that are federation specific.

Reply:
Agree. i thought instead of convuluting the existing function with lots of if conditions, why not define a separate copy_release_artifacts function for federation release.
I would update the existing function for federation release scenario.